src/cleartask.js

```diff
@@ -0,0 +1,11 @@
+import LocalStorage from './localStorage.js';
+
+const clearCompletedTasks = () => {
+  const localstore = new LocalStorage();
+  const filter = localstore.getTasks().filter((task) => !task.completed);
+  localstore.getTasks().length = 0;
+  localstore.getTasks().push(...filter);
+  localstore.saveTasks();
+};
+
+export default clearCompletedTasks;
```

Comment:
You have done a greate work in this project , but this file and method is not used it's better to remove it from your remote project to keep you code clean